### DISCUSSION TITLE
Memory leak with SKXamlCanvas

### DISCUSSION THREAD

**Discussion by LionelXN** [mono/SkiaSharp#2375] [Q&A]
I added several SKXamlCanvas with ItemsControl on the UI. When I remove them, I cannot completely release the requested memory, even I call GC. Collect()

**Comment by taublast** [comment]
Manually call YourView.Dispose(); for your SKXamlCanvas views, if this doesn't help for the memory call YourView.FreeBitmap(); before.
Also if you are on maui call YourView.Handler?.DisconnectHandler(); upon disposing
